tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ESNext",
```

Comment:
Otherwise an error throws with `es5`:
`Transforming object literal extensions to the configured target environment ("es5") is not supported yet`